src/main/java/de/uhd/ifi/se/decision/management/jira/persistence/DecisionGroupManager.java

```diff
@@ -154,18 +158,21 @@ public static boolean deleteInvalidGroups() {
 			String group = groupInDatabase.getGroup();
 			groups.add(group);
 		}
+		for (String group : groups) {
+			if (group.equals("High_Level") || group.equals("Medium_Level") || group.equals("Realization_Level")) {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5441305)